lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -143,12 +155,34 @@ for barclamp in $deactivation_list; do
 done
 
 
+### Stop chef-clients at the nodes, to ensure services shutdown in next step won't be started by chef
```

Comment:
While I'm nit-picking: I would probably only use `#` instead of `###` here (looking at the comment style, `###` is mostly used for big sections)